resources/i18n/en.json

```diff
@@ -115,6 +123,11 @@
     "item.metadata.dc.rights.uri.required": "Item metadata dc.rights.uri is required.",
     "item.metadata.dc.rights.uri.minLength": "Item metadata dc.rights.uri must have at least 4 characters.",
     "item.metadata.dc.rights.uri.maxLength": "Item metadata dc.rights.uri must have at most 512 characters.",
-    "item.metadata.dc.rights.uri.pattern": "Item metadata dc.rights.uri must be a FQDN."
+    "item.metadata.dc.rights.uri.pattern": "Item metadata dc.rights.uri must be a FQDN.",
     
+    "item.create.success": "Item ",
+    "item.create.error": "Error creating item ",
+
+    "create.success": " was created successfuly."
```

Comment:
That is convenient. Thanks. Would the brackets have to be escaped with the i18 messages being a json file?